Prepare for 1.9.0 release (#2307)
bottlerocket-os/bottlerocket: CHANGELOG.md

> # CHANGELOG 1.9.0

Reviewer: ```suggestion

```

# v1.9.0 (2022-07-28)

## OS Changes

### General

* SELinux policy now suppresses audit for tmpfs relabels ([#2222])
* Restrict permissions for /boot and System.map ([#2223])
* Remove unused crates growpart and servicedog ([#2238])
* Rust dependency updates ([#2303])
* Go dependency updates ([#2299])

> * Migrate to using `aws-sdk-rust` for first-party rust packages ([#2300])

Reviewer 2: Since the build tools are still using `rusoto` I might say first-party OS rust packages (just to make it more clear).

* New mount in host containers for system logs ([#2295])
* Add support for AWS China regions ( [#2224], [#2242], [#2247], [#2285])
* Apply strict mount options and enforce execution rules ([#2239])

> * Clean up uncommon way of disabling kernel config options ([#2290])

Reviewer 2: nit. It might be easier to follow if we say "Switch to a more commonly used syntax for disabling kernel config settings"

* Respect proxy settings when running setting generators ([#2227]) 
* Add `NET_CAP_ADMIN` to bootstrap containers ([#2266])
* Reduce log output for DHCP services ([#2260])
* Fix invalid kernel config options ([#2269])
* Improve support for container storage mounts ([#2240])
* Disable uncommon filesystems and network protocols ([#2255])
* Add support for blocking kernel modules ([#2274])
* Fix `ntp` service restart when settings change ([#2270])

> * Add kernel 5.15 sources ([#2226])

Reviewer: Missing link to PR

* Add “quiet” kernel parameter for AWS and VMware variants ([#2277])
* Defer `squashfs` mounts to later in the boot process ([#2276])
* Improve boot speed and rootfs size ([#2296])

### Kubernetes

* Make new instance types available ( [#2221] , thanks @cablespaghetti)
* Update Kubernetes versions ([#2230], [#2232], [#2262], [#2263], thanks @kschumy)

> * Add kubelet image GC threshold settings ([#2219])

Reviewer: Missing link to PR

### ECS

* Add iptables rules for ECS introspection server ([#2267])

### VMWare

* Remove `console=ttyS0` from kernel params ([#2248])

### Baremetal

> * Enable Mellanox modules in kernel-5.10 ([#2241])
> * Add bnxt module for Broadcom 10/25Gb network adapters in kernel-5.10 ([#2243])

Reviewer 2: ```suggestion
* Enable Mellanox modules in 5.10 kernel ([#2241])
* Add bnxt module for Broadcom 10/25Gb network adapters in 5.10 kernel ([#2243])
```
This is more consistent with our last set of release notes.

* Split out baremetal specific config options ([#2264])
* Add driver support for Cisco UCS platforms ([#2271])

## Build Changes

* Add Makefile targets to create and validate Boot Configuration ([#2189])

> * Create symlinks to images with friendly names ([#2215])

Reviewer: Missing PR link

* Add `start-local-vm` script ([#2194])
* Only build baremetal variant specific drivers for baremetal variants ([#2279])

> * Add the testsys cli and implent cargo make test for aws-k8s ([#2165])

Reviewer: nit: 
```suggestion
* Add testsys CLI and new test cargo-make tasks for testing aws-k8s variants ([#2165])
```

> * Enable the metal-dev build for the arm architecture ([#2272])

Reviewer: ```suggestion
* Enable the metal-dev build for the ARM architecture ([#2272])
```

## Documentation Changes

* Add NVIDIA ECS variant to README ([#2244])

> * Add documentation for metal variants ([#2205])

Reviewer: Missing PR links

* Add missing step in building packages guide ([#2259])
* Add quickstart for running Bottlerocket in QEMU/KVM VMs ([#2280])
* Address lints in README markdown caught by `markdownlint` ([#2283])

> 

Reviewer 2: ```suggestion
```
Extra line

# v1.9.0 (2022-07-28)

## OS Changes

### General

* SELinux policy now suppresses audit for tmpfs relabels ([#2222])

> * Restrict permissions for /boot and System.map ([#2223])
> * Remove unused crates growpart and servicedog ([#2238])

Reviewer: nits:
```suggestion
* Restrict permissions for `/boot` and `System.map` ([#2223])
* Remove unused crates `growpart` and `servicedog` ([#2238])
```

* Migrate to using `aws-sdk-rust` for first-party OS rust packages ([#2300])
* New mount in host containers for system logs ([#2295])

> * Add support for AWS China regions ( [#2224], [#2242], [#2247], [#2285])

Reviewer 3: ```suggestion
* Add support for AWS China regions ([#2224], [#2242], [#2247], [#2285])
```

> * Add support for AWS China regions ( [#2224], [#2242], [#2247], [#2285])

Reviewer 3: Maybe add an `### AWS` category, like we have for VMware, to collect the two AWS specific items.

* Apply strict mount options and enforce execution rules ([#2239])
* Switch to a more commonly used syntax for disabling kernel config settings ([#2290])
* Respect proxy settings when running setting generators ([#2227]) 
* Add `NET_CAP_ADMIN` to bootstrap containers ([#2266])
* Reduce log output for DHCP services ([#2260])
* Fix invalid kernel config options ([#2269])
* Improve support for container storage mounts ([#2240])
* Disable uncommon filesystems and network protocols ([#2255])
* Add support for blocking kernel modules ([#2274])
* Fix `ntp` service restart when settings change ([#2270])
* Add kernel 5.15 sources ([#2226])

> * Add “quiet” kernel parameter for AWS and VMware variants ([#2277])

Reviewer 3: nit: avoid smart quotes:
```suggestion
* Add "quiet" kernel parameter for AWS and VMware variants ([#2277])
```

* Defer `squashfs` mounts to later in the boot process ([#2276])
* Improve boot speed and rootfs size ([#2296])

### Kubernetes

> * Make new instance types available ( [#2221] , thanks @cablespaghetti)

Reviewer 3: ```suggestion
* Make new instance types available ([#2221] , thanks @cablespaghetti)
```

* Update Kubernetes versions ([#2230], [#2232], [#2262], [#2263], thanks @kschumy)
* Add kubelet image GC threshold settings ([#2219])

### ECS

* Add iptables rules for ECS introspection server ([#2267])

### VMWare

* Remove `console=ttyS0` from kernel params ([#2248])

> ### Baremetal

Reviewer 3: Maybe "Hardware" like last time, or "Hardware Support" ?
```suggestion
### Hardware
```

* Enable Mellanox modules in 5.10 kernel ([#2241])
* Add bnxt module for Broadcom 10/25Gb network adapters in 5.10 kernel ([#2243])
* Split out baremetal specific config options ([#2264])
* Add driver support for Cisco UCS platforms ([#2271])

## Build Changes

* Add Makefile targets to create and validate Boot Configuration ([#2189])
* Create symlinks to images with friendly names ([#2215])
* Add `start-local-vm` script ([#2194])
* Only build baremetal variant specific drivers for baremetal variants ([#2279])
* Add the testsys CLI and new cargo make tasks for testing aws-k8s variants ([#2165])

> * Enable the metal-dev build for the ARM architecture ([#2272])

Reviewer 3: The two metal notes can go into the bare metal / hardware category.

> * Update Rust dependency ([#2303])
> * Update Go dependency ([#2299])

Reviewer 3: ```suggestion
* Update Rust and Go dependencies ([#2303], [#2299])
```
